spec/models/news_tematica/news_tematica_spec.rb

```diff
@@ -1,43 +1,96 @@
 require 'spec_helper'
 
 describe NewsTematica do
-  let(:tematica) { FactoryGirl.create(:tematica) }
+  let(:tematica) { create(:tematica) }
 
   describe "calcula_fecha_desde" do
     before { allow(Time).to receive(:now).and_return(Time.parse('Feb 24 2013')) }
 
     it "debe devolver 'hace una semana' si no hay news anteriores de esta temática" do
-      news = FactoryGirl.build(:news_tematica)
+      news = build(:news_tematica)
       news.calcula_fecha_desde
       expect(news.fecha_desde).to eq Time.parse('Feb 17 2013')
     end
 
     it "debe devolver 'hace una semana' si la última news de esta temática es antigua" do
-      FactoryGirl.create(:news_tematica, tematica: tematica, fecha_hasta: Time.parse("Feb 03 2013"))
-      FactoryGirl.create(:news_tematica, tematica: tematica, fecha_hasta: Time.parse("Feb 01 2013"))
-      FactoryGirl.create(:news_tematica, fecha_hasta: Time.parse("Feb 22 2013"))
-      news = FactoryGirl.build(:news_tematica, tematica: tematica)
+      create(:news_tematica, tematica: tematica, fecha_hasta: Time.parse("Feb 03 2013"))
+      create(:news_tematica, tematica: tematica, fecha_hasta: Time.parse("Feb 01 2013"))
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.